spec/table_spec.rb

```diff
@@ -72,6 +72,48 @@
       expect(data_table.subtotal_calculations).to eq({["Star Wars"]=>[{:power_level=>{:sum=>145.0}}], ["Middle Earth"]=>[{:power_level=>{:sum=>9081.0}}]})
     end
 
+    it "should do sub-totaling starting with indexes > 0" do
+      data_table.group_by :world, level: 0
```

Comment:
`{level: 0}` is the [default value for the level param in the group by definition](https://github.com/veracross/data-table/blob/fix-unhandled-exception-for-nonzero-starting-subtotals/lib/data-table/table.rb#L173) so it can be removed.

```suggestion
      data_table.group_by :world
```

Reply:
I think it should have the border top by default in that case. Let's go ahead and add that.